### DISCUSSION TITLE
Limit file size of uploads

### DISCUSSION THREAD

**Discussion by jonplackett** [supabase/supabase#2971] [Questions] [answered]
Is there any way to limit file size of uploads to storage? 

Thanks!

**Comment by w3b6x9** [comment]
@jonplackett please email us at [support@supabase.io](support@supabase.io) with this request!

**Comment by fenos** [comment, marked as answer]
Here is a github issue https://github.com/supabase/storage-api/issues/134 describing this feature request, feel free to follow the progress there
